connection.go

```diff
@@ -206,9 +206,10 @@ type Connection struct {
 	healthCheckDone    chan struct{}
 	healthCheckHistory *healthHistory
 
-	// lastActivity is used to track how long the connection has been idle.
-	// (unix time, nano)
-	lastActivity atomic.Int64
+	// lastActivity{Recv,Send} is used to track how long the connection has been
+	// idle for the recieve and send connections respectively. (unix time, nano)
+	lastActivityRecv atomic.Int64
```

Comment:
nit: since Go uses `Read` and `Write`, and our functions mirror that name (`readFrames` and `writeFrames`) it may be better to use the same names here, `lastActivityRead` and `lastActivityWrite`

Reply:
Sure thing, done.